README.md

```diff
@@ -16,16 +16,58 @@ Build as any Maven-driven Java project:
 
 ## Usage
 
-*Step 1.* Start target JVM with with these additional flags:
 
-    $ java -XX:+UsePerfData -XX:+UnlockExperimentalVMOptions -XX:+ShenandoahRegionSampling ...
+#### Realtime
+*Step 1.* Start target JVM with these additional flags:
+
+    $ java -XX:+UseShenandoahGC -XX:+UsePerfData -XX:+UnlockExperimentalVMOptions -XX:+ShenandoahRegionSampling ...
 
 *Step 2.* Figure out the target JVM PID:
 
     $ jps
 
-*Step 3.* Attach the Visualizer:
+*Step 3a.* Run the Visualizer; Visualizer will attempt to detect a JVM running Shenandoah:
+    
+    $ java -jar visualizer.jar
 
-    $ java -Xbootclasspath/p:<path-to-tools.jar> -jar visualizer.jar local://<pid>
+*Step 3b.* Optionally attach the Visualizer using the -vm flag:
+  If using JDK8:
 
+    $ java -Xbootclasspath/p:<path-to-tools.jar> -jar visualizer.jar -vm local://<pid>
 `tools.jar` can usually be found at `$JAVA_HOME/lib`
+
+  If using JDK17+
+```bash
+  $ java --add-exports jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED -jar visualizer.jar
+```
+  Note that if you are using Intellij, you can also add these exports to the compiler (settings->javac) so that you can build and debug within the IDE.
+
+
+#### Saving JVM session
+Add this additional flag to an active JVM running Shenandoah:
+
+    $ -Xlog:gc+region=debug:<file name>::filesize=<Target byte size for log rotation>,filecount=<Number of files to keep in rotation>
```

Comment:
I am not convinced that unified logging is the right vehicle for recording visualizer region sampling information.

Reply:
We originally had a separate argument for configuring the log file, but as we started to add features to control the file's size and rotation it became clear that we were duplicating too much of the functionality already in unified logging - so we  scrapped it.